client/src/components/HostViews/Vote.vue

```diff
@@ -1,71 +1,53 @@
 <template>
-  <div>
-    <Tiebreaker :visible="showTiebreaker" />
-    <div style="position: absolute; top: 10%; left: 2%; z-index: 2">
-      <Clock title="Polls Close In" />
-    </div>
-    <div style="position: absolute; top: 10%; right: 2%; z-index: 2">
-      <Clock title="Polls Close In" />
-    </div>
-    <div class="center mt-5">
-      <div class="text-h4">Your Submissions</div>
-    </div>
-
-    <div class="center">
-      <div style="width: 75%;">
-        <v-row class="pa-4">
-          <v-col v-for="response in responses" :key="response.id">
-            <DisplayResponse :response="response" />
-          </v-col>
-        </v-row>
+  <div class="background-matte">
+    <header class="page-header center">
+      <div 
+        class="ma-3"
+        style="position: absolute; top: 0; right: 0;"
+      >
+        <Clock />
       </div>
-    </div>
-
-    <div class="center">
-      <div class="text-h4 mt-5">Live Voting Results</div>
-    </div>
-
-    <div class="bar-parent"> 
+      <h2 class="sub-title">It Is Time To</h2>
+      <h1 class="title">Vote</h1>
+    </header>
+    <div class="response-card-container mt-4">
       <div 
-        v-for="player in candidates" 
-        :key="player.clientId" 
-        class="center voter-bar" 
-        :style="`height: ${player.votes}%;`"
+        v-for="response in responses"
+        :key="response.player.clientId"
       >
-        <div class="text-p white--text name-tag">
-          {{ player.player.name }}
-        </div>
+        <ResponseCard
+          :response="response.response"
+          :player="response.player"
+          :showResponse="true"
+          :votePercentage="votePercentage(response.player.clientId)"
+        />
       </div>
     </div>
   </div>
 </template>
 
 <script>
-import HostMixin from "./HostMixin"
-import DisplayResponse from "./HostSubComponents/ResponseDisplayCard.vue"
-import Tiebreaker from "./HostSubComponents/TiebreakerCard.vue"
+import HostMixin from './HostMixin'
+import ResponseCard from './HostSubComponents/PlayerResponseCard.vue'
 import { Views } from '@/utils/Views'
 
 export default {
   mixins: [
     HostMixin
   ],
   components: {
-    DisplayResponse,
-    Tiebreaker
+    ResponseCard
   },
   data() {
     return {
       // stores ballots submitted by players
       ballotBox: {},
-      // candidate = { player: playerName, votes: totalVotes }
+      // candidate = { player: Player, votes: % of vote share }
```

Comment:
Is this dead code? If so, delete?

Reply:
not dead. I am just describing a candidate, which is inside the list of candidates